deploy/arax/scripts/start_app.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 # rename config file
-cp /configs/configv2.json /mnt/data/orangeboard/production/RTX/code/config_local.json
+cp /configs/config_secrets.json /mnt/data/orangeboard/production/RTX/code/config_local.json
```

Comment:
Confirm with Amy about the target config file name

Reply:
@pahmadi8740 - the destination here should be `config_secrets.json` as well. so the line should look like:
```
cp /configs/config_secrets.json /mnt/data/orangeboard/production/RTX/code/config_secrets.json
```
`config_local.json` shouldn't appear anywhere anymore.